File: 2013 ACM-ICPC North American Qualifier/oddities.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);
    
    for (int i = 0; i < n; i++) {
        int next;
        scanf("%d", &next);
        
        if (next & 1 == 1) {
            printf("%d is odd\n", next);
        } else {
            printf("%d is even\n", next);
        }
    }
    
    return 0;
}

File: 2013 ACM-ICPC North American Qualifier/oddities.cs
using System;

class Program {
    static void Main() {
        int n = int.Parse(Console.ReadLine());
        
        for (int i = 0; i < n; i++) {
            int next = int.Parse(Console.ReadLine());
            
            if ((next & 1) == 1) {
                Console.WriteLine($"{next} is odd");
            } else {
                Console.WriteLine($"{next} is even");
            }
        }
    }
}